src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
   // extract the json from the response
   const result = response.data.choices[0].text ?? ''
+
```

Comment:
### Line 26
## CodeGuard Suggestions
**Suggestion:** Combine line 28-32 and 36-39 into one block of code to make separating transpiling unnecessary
**Reason:** To make the code more concise and easier to maintain